### ISSUE TITLE
Add README example for v2 AWS SDK

### ISSUE THREAD

**Issue by nsearle** [raisenational/serverless-dynamodb#3]
It looks like your example code does not match up with your package dependency of the AWS SDK. In the example you are referencing the AWS SDK v3 usage `@aws-sdk` and not the v2 usage `aws-sdk` that your package.json is declaring. 

Here is a fitting v2 example: 

```
import AWS from "aws-sdk";

const client = new AWS.DynamoDB.DocumentClient({
  region: "localhost",
  endpoint: "http://0.0.0.0:8000",
  accessKeyId: "MockAccessKeyId",
  secretAccessKey: "MockSecretAccessKey",
});
```

I actually would prefer to use v3 of the aws sdk with this package. I'm guessing when built you were utilizing this in a library that already included the v3 sdk? 

And as a side note - if anyone is using Node 18+, you have to utilize a localized IP like `0.0.0.0` or `127.0.0.1` in both the dynamodb endpoint initialization config and the serverless yml `custom.dynamodb.start.host` config.

**Comment by domdomegg** [comment]
Hey @nsearle, thanks for the feedback :raised_hands:

I think I've identified three different parts to your issue - let me know if you see this differently. I've split these into separate threads so it's easier to continue to track and manage them longer term:

- (this issue) AWS SDK usage in the docs / your application
- #4: AWS SDK usage in serverless-dynamodb
- #5: Node 18 host configuration

**Comment by domdomegg** [comment]
To clarify the documentation, this library is compatible with any DynamoDB connector that supports DynamoDB Local, including the older `aws-sdk` and the newer `@aws-sdk/client-dynamodb`.

As such, I think the docs are currently correct. However, it still might be valuable to add your example as a guide to people using the older `aws-sdk`: I'm happy to add it myself or accept a PR on this. We could put these examples in different 'details' sections, similar to the way we've formatted them in the [README for `aws-ses-v2-local`](https://github.com/domdomegg/aws-ses-v2-local#setting-up-your-application).

**Comment by nsearle** [comment]
@domdomegg - I agree. After I posted, I actually realized that the underlying library dependency should not have an impact on the sdk version used at the higher level. 

It might be good to point out in the docs the implementation example on v2 and v3 like you suggested. If the PR has not been created by Monday, I will go ahead and take a stab at it!

Also I'm excited to see that an issue was created to migrate this library to v3!
